testdata/configtest.json

```diff
@@ -1446,6 +1446,40 @@
      "supportedCurrencies": ""
     }
    ]
+  },
+  {
+   "name": "Coinbene",
+   "enabled": true,
+   "verbose": false,
+   "websocket": false,
+   "useSandbox": false,
+   "restPollingDelay": 10,
+   "httpTimeout": 0,
+   "websocketResponseCheckTimeout": 0,
+   "websocketResponseMaxLimit": 0,
+   "websocketOrderbookBufferLimit": 0,
+   "httpUserAgent": "",
+   "httpDebugging": false,
+   "authenticatedApiSupport": false,
+   "authenticatedWebsocketApiSupport": false,
+   "apiKey": "Key",
+   "apiSecret": "Secret",
+   "apiUrl": "",
+   "apiUrlSecondary": "",
+   "proxyAddress": "",
+   "websocketUrl": "",
+   "availablePairs": "BTC/USDT",
+   "enabledPairs": "BTC/USDT",
+   "baseCurrencies": "",
+   "assetTypes": "SPOT",
+   "supportsAutoPairUpdates": true,
+   "configCurrencyPairFormat": {
+    "uppercase": true
```

Comment:
This is missing the "/" delimiter for config storage, as well as for the request currency pair format